runtime-modules/content/src/permissions/mod.rs

```diff
@@ -1,3 +1,20 @@
+/*
+* The following table summarizes the permissions in the content subsystem.
+* - Actor role as columns, controller account is Tx sender.
+* - operations on a given channel (=channel=) are rows, which are basically the guards to be
+*   implemented
+* - Entries are conditions to be verified / assertions
+*
+* |                       | *Lead*                   | *Curator*                | *Member*                | *Collaborator*                   |
+* |-----------------------+--------------------------+--------------------------+-------------------------+----------------------------------|
+* | *assets mgmt*         | channel.owner is curator | curator is channel.owner | member is channel.owner | collaborators in channel.collabs |
+* | *censorship mgmt*     | channel.owner is curator | curator is channel.owner | false                   | false                            |
```

Comment:
This is incorrect, the lead can censor all channels, curator can only censor member channels:
```suggestion
* | *censorship mgmt*     | true                     | channel.owner is member  | false                   | false                            |
```